contracts/Artists.sol

```diff
@@ -14,6 +14,9 @@ contract Artists is Ownable {
   struct Artist {
     string name;
     address wallet;
+    string nationality;
+    string birthYear;
+    string deathYear;
```

Comment:
Should we be storing birthYear and deathYear as uint256s instead? Personally I don't think it really matters, storing it as a string gives most flexibility for the user, but it may be more useful as an int in the future if we want to do any logic on these fields

Reply:
I have been using a string so far as easier to represent a deathYwar that doesn't exist yet. Something like "" or "N/A". Can change if you want but as harry said can be changed in the future if necessary